### PR TITLE
Fix the bug that when role count original value is unlimited, no cons…

### PR DESCRIPTION
**Pull request by ycheng-aa**
…traint.

JIRA: PDC-1111

### CODE REVIEW

File: pdc/apps/component/tests.py [hunk 1]
```diff
@@ -2787,6 +2787,33 @@ def test_constraint_to_contact_role_count_limit_change(self):
         response = self.client.patch(contact_role_url, data, format='json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_constraint_to_contact_role_count_limit_change_form_unlimited(self):
```
Comment (simozhan) [inline pdc/apps/component/tests.py]: form -> from

File: pdc/apps/component/tests.py [hunk 2]
```diff
@@ -2787,6 +2787,33 @@ def test_constraint_to_contact_role_count_limit_change(self):
...
+        # Add 3 contack for the role
```
Comment (simozhan) [inline pdc/apps/component/tests.py]: contack->contacts

**Comment by simozhan** [comment]
looks good to me. just some minor typo issues.

**Comment by bliuredhat** [comment]
:+1:

File: pdc/apps/component/tests.py (hunk 1 above)
Comment (ycheng-aa) [inline pdc/apps/component/tests.py]: Fixed.

File: pdc/apps/component/tests.py (hunk 2 above)
Comment (ycheng-aa) [inline pdc/apps/component/tests.py]: Fixed.

**Comment by xychu** [comment]
:+1:

**Comment by ycheng-aa** [comment]
Fixed typo and merged the commit.

**Pull request merged**